cmd/zpool/zpool_main.c

```diff
@@ -343,6 +357,8 @@ get_usage(zpool_help_t idx)
 		return (gettext("\treopen <pool>\n"));
 	case HELP_SCRUB:
 		return (gettext("\tscrub [-s] <pool> ...\n"));
+	case HELP_TRIM:
+		return (gettext("\ttrim [-s|-r <rate>] <pool> ...\n"));
```

Comment:
HELP_SCRUB needs the new -p something like "gettext("\ttrim [-s] | [-p] [-r <rate>] <pool> ...\n"));"

Reply:
Actually HELP_TRIM.